felix/deltatracker/delta_tracker.go

```diff
@@ -43,6 +43,16 @@ import (
 // keys that are in the dataplane map but not in the desired map.  The
 // pending maps are exposed via the IterPendingUpdates and IterPendingDeletions
 // methods.
+//
+// It's not recommended to use mutable objects in a DeltaTracker.  Updating a
```

Comment:
Can you be more precise?  E.g. "It's not recommended to pass a mutable object such as <X> to the <Y> API(s)."